tests/wpt/web-platform-tests/http/basic-auth-cache-test.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<meta charset="utf-8">
+<link rel="match" href="basic-auth-cache-test-ref.html">
+<html>
+  <div id="auth"> </div>
+  <div id="noauth"> </div>
+  <script type="text/javascript">
+    var authImg = '<img src="http://testuser:testpass@' + window.location.host + '/http/resources/securedimage.py">';
+    document.getElementById('auth').innerHTML = authImg;
+    setTimeout(function() {
```

Comment:
Thanks for the feedback! I've just incorporated the test changes in a new commit.